exchanges/orderbook/orderbook_test.go

```diff
@@ -506,31 +506,31 @@ func TestSorting(t *testing.T) {
 	}
 }
 
-func deploySliceOrdered() Tranches {
-	ts := make([]Tranche, 1000)
+func deploySliceOrdered() Levels {
+	ts := make([]Level, 1000)
```

Comment:
```suggestion
	l := make([]Level, 1000)
```
Or lvls and there's another one above it